modules/microshift-accessing-cluster-remotely-non-admin.adoc

```diff
@@ -25,6 +23,17 @@ Use the following procedure to access the {product-title} cluster from a remote
 [user@microshift]$ sudo firewall-cmd --permanent --zone=public --add-port=6443/tcp && sudo firewall-cmd --reload
 ----
 
+.Verify {product-title} is running
+
+. As `user@microshift`, verify that {product-title} is running by running the following command:
++
+[source,terminal]
+----
+[user@workstation]$ oc get all -A
```

Comment:
I don't think this verification step is needed here, but if we want it, it needs to be `[user@microshift]$ ...`.